### ISSUE TITLE
Bug when writing single Point while default_tags are provided

### ISSUE THREAD

**Issue by cboots** [influxdata/influxdb-client-python#116]
I was successfully using `write_api.write(record=point)` to write to InfluxDB, until I added default_tags to the write_api PointSettings.

Suddenly I saw a crash because the following bit of code doesn't have a case that handles `isinstance(record, Point)` case.

https://github.com/influxdata/influxdb-client-python/blob/5ddf643564bdd63aeb922a88e0a2c27d3a123de6/influxdb_client/client/write_api.py#L205

My workaround is to call `write_api.write(record=[point])` to fall back on the `List['Point']` use case which seems to work just fine.

Fix for this is easy, just add an `elif` case for record being a lone `Point`
```python
if isinstance(record, dict):
    record.get("tags")[key] = val
elif isinstance(record, Point):  # New case
    record.tag(key, val)  # New case
else:
    for r in record:
        ...
```

**Comment by bednar** [comment]
@cboots thanks, we will take a look

**Comment by bednar** [comment]
Hi @cboots,

The issue is fixed in [1.9.0 milestone](https://github.com/influxdata/influxdb-client-python/milestone/11). 

If you would like to use a dev version then install client via:

```pip install git+https://github.com/influxdata/influxdb-client-python.git@master```

Regards
